hnn_core/viz.py

```diff
@@ -61,7 +61,7 @@ def plt_show(show=True, fig=None, **kwargs):
 
 
 def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
-                show=True):
+                units=None, show=True):
```

Comment:
what happened to `scalings` parameter? I'm on the fence about whether users should be allowed scale the data. I think the data should only be scaled for viewing.